src/interactive.js

```diff
@@ -0,0 +1,39 @@
+import Methods from './utils.js';
+
+export default class Interactive {
+static toggleCompleted = (id, curStatus) => {
+  const todoList = Methods.getLocalStorageData();
+  todoList[id].completed = curStatus;
+  Methods.setLocalStorageData(todoList);
+  Methods.showTodoItems();
+}
+
+static addCheckEvent = () => {
+  document.querySelectorAll('.checkbox').forEach((box) => box.addEventListener('change', () => {
+    let id;
+    let curStatus;
+    if (box.id > 0) {
+      id = box.id - 1;
+    } else {
+      id = 0;
+    }
```

Comment:
For best JS practice (Don't repeat yourself), you can do a much neater if statement, for example, 
```
id = (box.id > 0) ? (box.id - 1) : 0
```